docker/Dockerfile

```diff
@@ -1,56 +1,57 @@
-### first stage - builder ###
-FROM golang:1.10 as builder
+# Global ARGs shared by all stages
+ARG DEBIAN_FRONTEND=noninteractive
+ARG GOPATH=/usr/local/go
 
-MAINTAINER Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
+### first stage - builder ###
+FROM debian:buster-slim as builder
 
-ENV HOME=/scratch
+ARG DEBIAN_FRONTEND
+ARG GOPATH
 
 # install debos build dependencies
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-    libglib2.0-dev \
-    libostree-dev \
-    && rm -rf /var/lib/apt/lists/*
+        ca-certificates \
+        golang-go \
+        gcc \
+        git \
+        libostree-dev \
+        libc6-dev && \
+    rm -rf /var/lib/apt/lists/*
 
+# Build debos
 RUN go get -d github.com/go-debos/debos/cmd/debos
-WORKDIR /go/src/github.com/go-debos/debos/
+WORKDIR $GOPATH/src/github.com/go-debos/debos/
 RUN GOOS=linux go build -a cmd/debos/debos.go
 
 ### second stage - runner ###
-FROM debian:stretch-slim as runner
+FROM debian:buster-slim as runner
 
-ARG DEBIAN_FRONTEND=noninteractive
+ARG DEBIAN_FRONTEND
```

Comment:
Why unseting this one ?

Reply:
This doesn't unset DEBIAN_FRONTEND but allows the stage to use the global ARG